apm-agent-plugins/pom.xml

```diff
@@ -58,6 +58,7 @@
         <module>apm-rabbitmq</module>
         <module>apm-okhttp-test</module>
         <module>apm-cassandra</module>
+        <module>apm-struts-plugin</module>
```

Comment:
[question] would it make sense to have a separate module for struts1 and struts2 ? If so, should we rather have `struts2` in the module name ?